pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -215,11 +216,12 @@ func (h *mtuHandler) RemoteEndpointRemoved(endpoint *submV1.Endpoint) error {
 	return nil
 }
 
-func extractSubnets(endpoint *submV1.EndpointSpec) []string {
+func extractIPv4Subnets(endpoint *submV1.EndpointSpec) []string {
 	subnets := make([]string, 0, len(endpoint.Subnets))
 
 	for _, subnet := range endpoint.Subnets {
-		if !strings.HasPrefix(subnet, endpoint.PrivateIP+"/") {
```

Comment:
So we no longer need this check?

Reply:
No. I don't think we require this check. Prior to pushing the PR, I discussed with @aswinsuryan (original author of the code) to see if there is a use-case behind this check.

Reply:
I think  reused a method from here at that time and it may not be valid here  given the IPs that an endpoint can take,

[1] https://github.com/submariner-io/submariner/blob/4eef0296c913b8d4bb9f7be33edbcdcb63891ecb/pkg/cable/libreswan/libreswan.go#L288

Reply:
Thanks for confirming @aswinsuryan